composer.json

```diff
@@ -91,6 +92,7 @@
         "sprain/swiss-qr-bill": "^4.3",
         "square/square": "30.0.0.*",
         "stripe/stripe-php": "^12",
+        "symfony/brevo-mailer": "^7.0",
```

Comment:
I haven't tested this, but I would assume we would want ^6.0 here as the other dependencies are ^6.0 based.

Reply:
I changed it thanks